cert_manager/manifest.json

```
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a3b15f7a-5edb-4d8b-8e74-84cabf38b7e7",
```

https://github.com/DataDog/integrations-extras/blob/185c1b6cf1d896579e7a1abe695ebd75b0ef98c8/cert_manager/manifest.json#L3

```suggestion
  "app_uuid": "d8bac6db-8cf7-49ca-a4b8-643714fbc7b9",
```